cdap-app-templates/cdap-etl/cdap-etl-core/src/main/resources/etl.properties

```diff
@@ -13,5 +13,5 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 #
-etl.batch.plugin.name=etlBatch
-etl.realtime.plugin.name=etlRealtime
+etl.batch.plugin.name=${etl.batch.plugin.name}
```

Comment:
The value is set in maven pom property. The maven-resources-plugin will do the substitution. By doing this, we have one place to define the names.